contrib/diurnal/README.md

```diff
@@ -0,0 +1,39 @@
+## Diurnal Controller
+This controller manipulates the number of replicas maintained by a replication controller throughout the day based on a provided list of times of day and replica counts. It should be run under a replication controller that is in the same namespace as the replication controller that it is manipulating.
+
+Instead of providing replica counts and times of day directly, you may use a script like the one below to generate them using mathematical functions.
+
```

Comment:
Add an example of what a list of these looks like.

Add a link to the example YAML